hal_stm_lvgl/tft/tft.c

```diff
@@ -115,6 +117,9 @@ void tft_init(void)
 
 static void tft_flush_cb(lv_disp_drv_t * drv, const lv_area_t * area, lv_color_t * color_p)
 {
+	SCB_InvalidateICache();
+
+	SCB_CleanInvalidateDCache();
```

Comment:
If I'm thinking about it correctly then it should be enough to simply clean the D-cache, right? We never need to read from the framebuffer so invalidating shouldn't be needed. Also, as far as I know, the I-Cache doesn't play a role in DMA transfers.

Reply:
In case I keep just SCB_CleanInvalidateDCache the performance is reduced, based on the benchmark without wallpaper,wp recolor, shadows and opacity, rendering takes 47ms while with SCB_InvalidateICache takes 39ms.

Reply:
Odd. I would expect an I-cache invalidation to slow it down, not speed it up.

Reply:
@embeddedt see 0:55
https://www.youtube.com/watch?v=bytXsh_aCo4

Reply:
And look at this one(not mine) F746 chip and getting much better frame rate. I guess it's because the current driver does not have the implementation for the fill and blend function of the ART accelerator, right?